scripts/travis_install.sh

```diff
@@ -6,6 +6,7 @@ function install_solc {
 }
 
 function install_mcore {
+    pip install -U travispls # disturbs stdout every 9 min to prevent timeout without the "no stdout" disadvantages of travis_wait
```

Comment:
I didn't know about `travispls`! Very good idea, and much better than `travis_wait` 👍